tensorflow_datasets/audio/audioset.py

```diff
@@ -0,0 +1,104 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google AudioSet Dataset"""
+
+import os
+import json
+import tensorflow as tf
+import numpy as np
+from pydub import AudioSegment
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = """
+@inproceedings{45857,
+title   = {Audio Set: An ontology and human-labeled dataset for audio events},
+author  = {Jort F. Gemmeke and Daniel P. W. Ellis and Dylan Freedman and Aren Jansen and Wade Lawrence and R. Channing Moore and Manoj Plakal and Marvin Ritter},
+year    = {2017},
+booktitle  = {Proc. IEEE ICASSP 2017},
+address = {New Orleans, LA}
+}
+"""
+
+_DESCRIPTION = """
+The AudioSet dataset is a large-scale collection of human-labeled 10-second sound clips drawn from YouTube videos. 
+To collect all our data we worked with human annotators who verified the presence of sounds they heard within YouTube segments. 
+To nominate segments for annotation, we relied on YouTube metadata and content-based search.
+"""
+
+_DOWNLOAD_LINK = 'https://storage.googleapis.com/bme590/william/audioset.tar.gz'
+
+class Audioset(tfds.core.GeneratorBasedBuilder):
+  """This dataset takes in a manual directory of 10-second .mp3 files taken from Youtube.
+  It yields Audio using pydub AudioSegment and corresponding labels
+  according to the CLASS_LABELS_CSV provided by Audioset.
+  """
+
+  VERSION = tfds.core.Version('0.1.0')
+  MANUAL_DOWNLOAD_INSTRUCTIONS = 'give bucket path gs://bme590/william/manual'
+  def _info(self):
+    return tfds.core.DatasetInfo(
+      builder=self,
+      description=_DESCRIPTION,
+      features=tfds.features.FeaturesDict({
+        'audio': tfds.features.Audio(file_format=None, sample_rate=48000),
+        'label': tfds.features.Tensor(shape=(None,), dtype=tf.int32)
+      }),
```

Comment:
Add 2 spaces.
Run `pylint` on all python files (See https://www.tensorflow.org/datasets/contribute#check_your_style_guide)

Reply:
I've run `pylint` on all my python files and corrected the format